src/sst/core/testingframework/sst_unittest.py

```diff
@@ -234,7 +234,7 @@ def get_test_runtime_sec(self):
 
     def run_sst(self, sdl_file, out_file, err_file=None, set_cwd=None, mpi_out_files="",
                 other_args="", num_ranks=None, num_threads=None, global_args=None,
-                timeout_sec=60):
+                timeout_sec=60, expected_rc=0):
```

Comment:
Need to update the args section of the method's docstring explaining expected_rc 
```
global_args (str): Global Arguments provided from test engine args
expected_rc (int): The expected return code from the SST run
```

Need to do a check on expected_rc to verify its a valid data type
```
        if not (isinstance(timeout_sec, (int, float)) and not isinstance(timeout_sec, bool)):
            raise ValueError("ERROR: Timeout_sec must be a postive int or a float")
        check_param_type("expected_rc", expected_rc, int)  <-- add this ine
```

Reply:
Added